prow/prowjobs/kubeflow/kubeflow-periodics.yaml

```diff
@@ -0,0 +1,153 @@
+periodics:
+- name: kubeflow-periodic-0-6-branch
+  interval: 8h
+  labels:
+    preset-service-account: "true"
```

Comment:
This winds up injecting some ENV, volumes, volume mounts

https://github.com/kubernetes/test-infra/blob/8cc13001c254cef1d88b13f5f9d5d1eaf05f84d0/config/prow/config.yaml#L475-L491

These don't yet exist in this cluster. https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/config.yaml


Do you know what you are using this service account to do?

Reply:
I don't but I can find out but per the comment here https://github.com/kubernetes/test-infra/issues/14343#issuecomment-534075549 can we just use our own cluster to run these tests? If so how do we go about doing that?

If not, how do we get these ENV, volumes, and mounts into the oss-test-infra cluster?

Reply:
Yes, creating your own cluster is ideal. Can you talk to @clarketm about how to do this? He's been working on tooling to help you create build clusters.

Reply:
I added the `preset-service-account` preset #111 in case it is needed.